otterdog/eclipse-openbsw.jsonnet

```diff
@@ -22,6 +22,7 @@ orgs.newOrg('eclipse-openbsw') {
       delete_branch_on_merge: true,
       has_issues: true,
       allow_merge_commit: false,
+      allow_squash_merge: true,
```

Comment:
squash merge is already enabled by default, see https://otterdog.eclipse.org/projects/automotive.openbsw/repos/openbsw#settings

Reply:
oh...I see...thanks!